src/main/target/system_stm32f4xx.c

```diff
@@ -540,7 +540,17 @@ void SystemInit(void)
 
   /* Configure the Vector Table location add offset address ------------------*/
 #ifdef VECT_TAB_SRAM
-  SCB->VTOR = SRAM_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal SRAM */
+  extern uint8_t isr_vector_table_flash_base;
+  extern uint8_t isr_vector_table_base;
+  extern uint8_t isr_vector_table_end;
+  uint32_t *vecTableSrc = (uint32_t *)&isr_vector_table_flash_base;
```

Comment:
Explicit `memcpy`.

Reply:
It will (probably) work, but system is not fully initialized yet. It is a bit cleaner to use explicit copy here.

Reply:
It will work. It's not cleaner as is.

Reply:
I'm opted for `memcpy`.